DRCODEOWNERS

```diff
@@ -42,6 +42,7 @@
 /public_dropin_environments/python3_xgboost                       @datarobot/custom-models @datarobot/core-modeling
 /public_dropin_environments/r_lang                                @datarobot/custom-models @datarobot/core-modeling
 /public_dropin_gpu_environments                                   @datarobot/tracking-agent @datarobot/custom-models
+/public_dropin_gpu_environments/python311_genai                   @datarobot/buzok
```

Comment:
@yakov-g with only having a single team reviewing, can we change this repo to only require a single reviewer?

Reply:
We historically required two because it is public repo. Just to have another pair of eyes.
Generally we could remove, but I'm not sure.

Reply:
@baekdahl @mkavf Maybe you could update ownership for it accordingly. And for the original genai env as well.